corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/views.js

```diff
@@ -208,19 +208,39 @@ hqDefine("cloudcare/js/formplayer/menus/views", function () {
         return caseListLayout;
     };
 
+    const getMapScrollOffset = function (addressMap, smallScreenEnabled) {
```

Comment:
This is less a specific suggestion and more something to consider if we're making this a more general-purpose function. There might be a reason to pass in `addressMap` itself too, maybe it was a concern with timing and load/attach order.
```suggestion
    const getScrollTopOffset = function (smallScreenEnabled, isFullscreen = false) {
```